app.ts

```diff
@@ -46,14 +46,16 @@ const redis = !!ENVIRONMENT.REDIS_URL ? new Redis(ENVIRONMENT.REDIS_URL) : null;
  * @param {Event} targetEvent リプライ対象のイベント
  * @param {number} created_at 作成するイベントの日時
  */
-const composeReplyPost = (content: string, targetEvent: Event, created_at: number = currUnixtime() + 1) => {
+const composeReplyPost = (content: string, targetEvent: Event) => {
+  const tags = [];
+  tags.push(['e', targetEvent.id], ['p', targetEvent.pubkey])
+  if (targetEvent.kind == 42)
+    for (let tag of targetEvent.tags.filter((x: any[]) => x[0] === 'e')) tags.push(tag)
+  const created_at: number = targetEvent != null ? targetEvent.createdAt + 1 : currUnixtime() + 1
```

Comment:
Initially, the logic used the creation time +1 if created_at was included in the argument, and the current time +1 if not. This code would not work properly in [reminder notification section](https://github.com/ikuradon/823chan/blob/55425bb338349839ec40245428c66c6d826208a4/app.ts#L1789).
As a countermeasure, it would be better to set created_at in [the original event creation part](https://github.com/ikuradon/823chan/blob/55425bb338349839ec40245428c66c6d826208a4/app.ts#L1802) to the current time (`currUnixtime()`).

Reply:
fixed